TS.fsx

```diff
@@ -506,7 +523,7 @@ let EmitStaticInterface flavor (i:Browser.Interface) =
         EmitConstants i
         EmitEventHandlers prefix i
         EmitIndexers EmitScope.StaticOnly i
-
+        emitAddedConstructor ()
```

Comment:
This happens when the spec wrongly marked an interface as pure static, and a constructor is added in the json files. For example Microsoft/TypeScript#2583